### PR TITLE
test: add a meta test to make sure that v4/objects/ files are imported

### PR DESCRIPTION
**Pull request by JohnVillalovos**
Add a test to make sure that all of the `gitlab/v4/objects/` files are
imported in `gitlab/v4/objects/__init__.py`

### CODE REVIEW

File: tests/meta/test_v4_objects_imported.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    print(os.getcwd())
+    print(V4_OBJECTS_DIR)
+    object_files: Set[str] = set()
+    for filename in sorted(os.listdir(V4_OBJECTS_DIR)):
+        if not filename.endswith(".py"):
+            continue
+        if filename in ("__init__.py",):
+            continue
+        root, _ = os.path.splitext(filename)
+        object_files.add(root)
```
Comment (nejch) [inline tests/meta/test_v4_objects_imported.py:32]: I think this part could also just be:
```python
import pkglib

import gitlab.v4.objects

modules =  [module.name for module in pkgutil.iter_modules(gitlab.v4.objects.__path__)]
```

Might need fewer constants and so on at the top then.

Comment (JohnVillalovos) [reply tests/meta/test_v4_objects_imported.py]: Done. Mostly like that.

**Review by nejch — COMMENTED** [review]
I think we can go ahead and merge this with an xfail if you like and then fix that when the access token PR is finished, WDYT? Just pasted some ideas also.

File: tests/meta/test_v4_objects_imported.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import os
+import pkgutil
+from typing import Set
+
+import gitlab.v4.objects
+
+
+def test_verify_v4_objects_imported() -> None:
+    assert len(gitlab.v4.objects.__path__) == 1
+    v4_objects_init_file = os.path.join(gitlab.v4.objects.__path__[0], "__init__.py")
```
[the remark this answers is not in the document]
Comment (JohnVillalovos) [reply tests/meta/test_v4_objects_imported.py:15]: Done

File: tests/meta/test_v4_objects_imported.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    init_files: Set[str] = set()
+    with open(v4_objects_init_file, "r") as in_file:
+        for line in in_file.readlines():
+            if line.startswith("from ."):
+                init_files.add(line.rstrip())
```
[the remark this answers is not in the document]
Comment (JohnVillalovos) [reply tests/meta/test_v4_objects_imported.py:21]: I tried it but it ended up being three lines after black formatted it. So I left it as is.

**Review by nejch — APPROVED** [review]

**Pull request merged**